dask_glm/logistic.py

```diff
@@ -288,97 +288,94 @@ def identity(x, t):
     return beta
 
 
-def logistic_regression(X, y, alpha, rho, over_relaxation, max_iter=100):
-    N = 5  # something to do with chunks
-    (m, n) = X.shape
-
-    z = np.zeros([n, N])
-    a = np.zeros([n, N])  # to become u
-
-    # why convert this to a dask array?
-    beta_old = da.from_array(np.zeros([n, N]), chunks=(2, 1))
-    beta = np.zeros((2, 5))
-    u = da.from_array(a, chunks=(2, 1))
-    MAX_ITER = max_iter
-    ABSTOL = 1e-4
-    RELTOL = 1e-2
-
-    for k in range(MAX_ITER):
-        beta_x = y.map_blocks(local_update, X, beta_old.T, z[:, 0], u.T, rho,
-                              chunks=(5, 2), dtype=float).compute()
-        beta[0, :] = beta_x[0::2].ravel()
-        beta[1, :] = beta_x[1::2].ravel()
-        beta_hat = over_relaxation * beta + (1 - over_relaxation) * z
+def admm(X, y, lamduh=0.1, rho=1, over_relax=1,
+         max_iter=100, abstol=1e-4, reltol=1e-2):
+
+    chunks = X.npartitions
+    (n, p) = X.shape
+    XD = X.to_delayed().flatten().tolist()
+    yD = y.to_delayed().flatten().tolist()
+
+    z = np.array([np.zeros(p) for i in range(chunks)])
+    u = np.array([np.zeros(p) for i in range(chunks)])
+    betas = np.array([np.zeros(p) for i in range(chunks)])
```

Comment:
`np.zeros((chunks, p))`

Also, slight preference to use the term `nchunks` to avoid colliding with dask.array's use of chunks, which has a different meaning.